src/main/resources/assets/computercraft/lua/rom/help/credits.txt

```diff
@@ -8,3 +8,16 @@ Visit http://www.minecraftu.org and http://www.deepspace.me/space-cadets to find
 
 Join the ComputerCraft community online at http://www.computercraft.info
 Follow @DanTwoHundred on Twitter!
+
+GitHub Contributors:
```

Comment:
I think this should just say "Contributors". That'd leave room for people who didn't contribute directly via Git.